assets/queries/common/passwords_and_secrets/regex_rules.json

```diff
@@ -14,7 +14,8 @@
           "regex": "['\"]?AllowUsersToChangePassword['\"]?\\s*[:=]\\s*['\"]?([A-Za-z0-9/~^_!@&%()=?*+-.]{4,})['\"]?"
         }
       ],
-      "specialMask": "(?i)['\"]?password['\"]?\\s*[:=]\\s*"
+      "multiline": false,
```

Comment:
Can't we expect the multiline value to be set as false by default?

Reply:
we can, I used gpt to write the script and it decided to set it to false 😄, done [here](https://github.com/Checkmarx/kics/pull/6858/commits/11a7396a3ef712a552c18ffe0eff2ba0b210f6fc)